app/src/main/kotlin/co/netguru/baby/monitor/client/feature/noisedetection/NoiseDetector.kt

```diff
@@ -5,23 +5,23 @@ import io.reactivex.Single
 import timber.log.Timber
 import javax.inject.Inject
 import kotlin.math.log10
+import kotlin.math.roundToInt
 import kotlin.math.sqrt
 
 class NoiseDetector @Inject constructor() {
-    fun processData(data: ShortArray, size: Int) = Single.fromCallable<Double> {
+    fun processData(data: ShortArray) = Single.fromCallable<Int> {
         var decibels = 0.0
-        if (size > 0) {
+        if (data.isNotEmpty()) {
             var totalSquared = 0.0
-            for (i in 0 until size) {
-                val soundBit = data[i]
+            for (soundBit in data) {
                 totalSquared += (soundBit * soundBit).toDouble()
             }
 
-            val quadraticMeanPressure = sqrt(totalSquared / size)
+            val quadraticMeanPressure = sqrt(totalSquared / data.size)
             decibels = FORMULA_CONSTANT * log10(quadraticMeanPressure)
         }
         Timber.d("Sound Db: $decibels")
-        return@fromCallable decibels
+        return@fromCallable decibels.roundToInt()
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/92c7fea843a9bd05df3c694ba8c05996f0ca5e22/app/src/main/kotlin/co/netguru/baby/monitor/client/feature/noisedetection/NoiseDetector.kt#L24"></span>Expression with labels increase complexity and affect maintainability.</td>
    </tr>
  </tbody>
</table>